src/en/controllers.md

```diff
@@ -73,6 +73,16 @@ Common tasks are summarized below.
 
 
 
+^# Configure a controller
+
+   There are several settings which can be adjusted from their defaults, such
+   as whether the controller will record auditing information. Configuration
+   must happen as the controller is bootstrapped.
+
+   This is a complex subject. See [Configuring controllers](./controllers-config.html).
```

Comment:
Can we instead say:

This topic is covered in more depth in the [configuring controllers section]